api/views/problemlist.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework import serializers
+from rest_framework.generics import RetrieveAPIView
+
+from problem.models import Problem
+
+
+class ProblemListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Problem
+        fields = ["problem_list_info"]
+
+    problem_list_info = serializers.SerializerMethodField()
+
+    def get_problem_list_info(self, problem):
+        problems = Problem.objects.all().order_by("id")
+        ids = [_id[0] for _id in problems.values_list("id")[:]]
+        titles = [title[0] for title in problems.values_list("title")[:]]
+        id_titles = []
+        space_between = ".   "
+        for i in range(len(ids)):
```

Comment:
I remember `str(problem)` will immediately achieve what you want. Please check.

Reply:
@Roooooobin